packages/quantic/force-app/main/default/lwc/quanticFacetValue/quanticFacetValue.js

```diff
@@ -3,11 +3,13 @@ import {LightningElement, api} from 'lwc';
 export default class QuanticFacetValue extends LightningElement {
   /** @type {import("coveo").FacetValue} */
   @api item;
+  /** @type {boolean} */
+  @api isChecked;
 
   /**
    * @param {InputEvent} evt
    */
-  onSelect(evt) {
+  facetValueClick(evt) {
```

Comment:
Just curious why the renaming? If we rename could we still keep an "onFacetValueClick" or "handleFacetValueClick" syntax so we know it's an event handler right away?

Reply:
I think ill remove that from the PR, it was to allow clicking on the whole facet value line instead of just on the checkbox, but it's out of scope and I'll make another general facets improvement Jira and PR for this.